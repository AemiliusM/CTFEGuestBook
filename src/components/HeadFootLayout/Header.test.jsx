import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

it('should render the header', () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
