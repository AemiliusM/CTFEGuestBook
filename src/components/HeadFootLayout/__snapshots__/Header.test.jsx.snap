// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header 1`] = `
<div>
  <header>
    This is a Header!
  </header>
</div>
`;
